### DISCUSSION TITLE
word pairs experiment/ two part trial

### DISCUSSION THREAD

**Discussion by Altr37** [jspsych/jsPsych#1336] [Q&A] [answered]
Hi,

I was hoping you can help me figure out how to create an experiment where the trials consisted of two parts, like in a word pair experiment: word 1 is presented on screen and disappears, then word 2 presented and response is required.
Response is required only for w2, but it must be preceded by its own w1.

How to create those pairs, such that I can randomize word pair trials while keeping w1-w2 appropriately attached (presented sequentially)?

**Comment by jodeleeuw** [comment, marked as answer]
Hi @Altr37 ,

This is a perfect use case for timeline variables. The [documentation](https://www.jspsych.org/overview/timeline/#timeline-variables) goes through a pretty similar use case, but with image/word pairs. Feel free to follow up here if you need additional clarification.

**Reply by Altr37** [reply]
Thank you!
